modules/cassandra/src/main/java/org/apache/ignite/cache/store/cassandra/utils/session/CassandraSessionImpl.java

```diff
@@ -134,7 +134,7 @@ else if (CassandraHelper.isHostsAvailabilityError(e))
                     else if (!CassandraHelper.isPreparedStatementClusterError(e))
                         throw new IgniteException(errorMsg, e);
 
-                    if (log != null && !CassandraHelper.isPreparedStatementClusterError(e))
+                    if (!CassandraHelper.isPreparedStatementClusterError(e))
```

Comment:
It will produce NullPointerException if Ignite was started with Basic configuration without Spring support added (see the same comment above)